docs/current/742989-cookbook.md

```diff
@@ -625,6 +625,44 @@ Set the Hashicorp Vault URI, namespace, role and secret identifiers as host envi
 
 [Learn more](./guides/723462-use-secrets.md)
 
+
+## Error handling
+
+### Aborting with unhandled error
+
+The following code listing demonstrates how to abort the script with an unhandled error, which allows debugging with the stack trace of the crash during development.
+
+```go file=./cookbook/snippets/error-handling/unhandled/main.go
+```
+
+### Aborting gracefully
+
+The following code listing demonstrates how to abort the script without crashing.
+
+```go file=./cookbook/snippets/error-handling/aborting/main.go
+```
+
+### Exit code and unexpected errors
+
+The following code listing demonstrates how to handle a non-zero exit code from running a command, with several use cases:
+- Difference between “test failed” and “failed to test”
+- Handle a specific exit code value
+- Handle an exec error without checking for the exit code
+- Handle an exec error without propagating
```

Comment:
```suggestion
- Handle an error without propagating it
```

Reply:
Here it loses a bit more meaning. The use case is to not propagate an ExecError, but let other errors propagate.

Reply:
See if mod makes it clearer?